build/package.js

```diff
@@ -0,0 +1,151 @@
+const { resolve } = require('path')
+const { spawnSync } = require('child_process')
+const consola = require('consola')
+const { readFileSync, existsSync, readJSONSync, writeFileSync, copySync } = require('fs-extra')
+const builtin = require('./builtin.json')
+
+module.exports = class Package {
```

Comment:
I love how clean this is -- could we get this working with ESM syntax though?

Reply:
Good idea. We can use `node -r esm`